test/extensions/filters/network/common/fuzz/uber_per_writefilter.cc

```diff
@@ -10,21 +10,8 @@ std::vector<absl::string_view> UberWriteFilterFuzzer::filterNames() {
   // Will extend to cover other network filters one by one.
   static std::vector<absl::string_view> filter_names;
   if (filter_names.empty()) {
-    const auto factories = Registry::FactoryRegistry<
-        Server::Configuration::NamedNetworkFilterConfigFactory>::factories();
-    const std::vector<absl::string_view> supported_filter_names = {
-        NetworkFilterNames::get().ZooKeeperProxy, NetworkFilterNames::get().KafkaBroker,
-        NetworkFilterNames::get().MongoProxy, NetworkFilterNames::get().MySQLProxy
-        // TODO(jianwendong) Add "NetworkFilterNames::get().Postgres" after it supports untrusted
-        // data.
-    };
-    for (auto& filter_name : supported_filter_names) {
-      if (factories.contains(filter_name)) {
-        filter_names.push_back(filter_name);
-      } else {
-        ENVOY_LOG_MISC(debug, "Filter name not found in the factory: {}", filter_name);
-      }
-    }
+    filter_names = Registry::FactoryRegistry<
```

Comment:
nit: please add a comment similar to the one you added in uber_per_readfilter.cc